### PR TITLE
Adds i18n with support for English UK & US

### PR DESCRIPTION
**Pull request by clementmouchet**
Hi @olup as discussed I've added support for English UK & US for the UI & tested the TTS as well.

### CODE REVIEW

File: src/components/LanguageSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          .then(() => setSelected(item))}
+      key={item.locale}
+    >
+      {item.variant}
+    </Menu.Item>
+  ));
+
+  return (
+    <Menu
+      onOpen={() => setOpened(true)}
```
Comment (olup) [inline src/components/LanguageSelector.tsx:34]: I don't believe you need a controlled menu here, do you ?

Comment (clementmouchet) [reply src/components/LanguageSelector.tsx:34]: I'm new to mantine, but I thought it was a good way to get it to manage selection. I'll have a look at other components to see if there's anything more stuitable

Comment (clementmouchet) [reply src/components/LanguageSelector.tsx:34]: btw, le me know if you have anything in mind :)

File: src/i18n/i18n.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import {initReactI18next} from 'react-i18next';
+import LanguageDetector from "i18next-browser-languagedetector";
+import en from './locales/en-GB/translation.json';
+import fr from './locales/fr-FR/translation.json';
+
+const isProduction = process.env.NODE_ENV === 'production';
+
+i18n.use(initReactI18next)
+    .use(LanguageDetector)
+    .init({
```
Comment (olup) [inline src/i18n/i18n.ts:11]: do we want to set up the default language (which should probably be french as it has the broadest French community) ?

**Comment by olup** [comment]
Awesome, thanks for pushing this. If you want to do lunii-admin-web later :-D

**Comment by clementmouchet** [comment]
That's what I had initially but I think the LanguageDetector should take care of that, as most browsers will be supported. Happy to set it as default though.
My laptop and VMs are all in English, so I have to confess I didn't try. What happens when you run it?

File: src/i18n/i18n.ts (hunk 2 above)
Comment (clementmouchet) [reply src/i18n/i18n.ts:11]: That's what I had initially but I think the LanguageDetector should take care of that, as most browsers will be supported. Happy to set it as default though.
My laptop and VMs are all in English, so I have to confess I didn't try. What happens when you run it?

**Comment by clementmouchet** [comment]
Yep, I'll take a look at lunii-admin-web when I have a chance.
Before that I'd like to push another PR to prepend a bit of silence before the generated TTS so it doesn't chop the start when  playing on the lunii

File: src/components/LanguageSelector.tsx (hunk 1 above)
Comment (olup) [reply src/components/LanguageSelector.tsx:34]: Oh, I juste mean you can remove the open state and the onOpen on close props it will manage that auto

**Comment by olup** [comment]
Hey, thanks for this I got delayed but it's merged. For the silence I wonder if this should be added here or globally in the conversion step of lunii admin web. But i'll be glad if you want to give it a try on any repo.

**Pull request merged**